### PR TITLE
Updated the fields on the Deal cards

### PR DESCRIPTION
**Pull request by kapildave07**
## Description

Updated the fields on the Deal cards fixed UI 

## Changes

- UI have been fixed

## Attached Links

1. Jira ticket: https://investax.atlassian.net/browse/IXSPD1-308
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/state/launchpad/types.ts [hunk 1]
```diff
@@ -167,10 +167,10 @@ export enum InvestmentStatus {
...
 
 export enum OfferInvestmentStructure {
-  equity = 'equity',
-  debt = 'debt',
-  hybrid = 'hybrid',
-  other = 'others',
+  equity = 'Equity',
+  debt = 'Debt',
+  hybrid = 'Hybrid',
+  other = 'Others',
```
Comment (thi-investax) [inline src/state/launchpad/types.ts:173]: @kapildave07 , could you please check if these enums are being used in API call? if you change here api calls may not work

File: src/components/Launchpad/InvestmentCard/index.tsx [hunk 2]
```diff
@@ -291,16 +308,14 @@ const InvestmentCardDescription = styled.div`
...
 
   transition: height 0.3s ease-in-out, opacity 0.2s ease-out 0.1s;
 
-  z-index: 10;
-
   ${(props) => props.show && `margin: 0.5rem -1.5rem;`}
 
   border-top: 1px solid ${(props) => props.theme.launchpad.colors.border.default};
-  border-bottom: 1px solid ${(props) => props.theme.launchpad.colors.border.default};
+  // border-bottom: 1px solid ${(props) => props.theme.launchpad.colors.border.default};
```
Comment (leonthuongto) [inline src/components/Launchpad/InvestmentCard/index.tsx:318]: should to remove haha

**Review by leonthuongto — APPROVED** [review]

**Comment by thi-investax** [comment]
image
do we have the currency @kapildave07 ?

**Comment by kapildave07** [comment]
@thi-investax we have investingTokenSymbol not Screenshot 2024-07-02 at 11 42 25 AM

**Review by thi-investax — APPROVED** [review]

**Pull request merged**